### DISCUSSION TITLE
Peak2Gene Legend Range

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#772] [Questions / Documentation]
<img width="286" alt="Screen Shot 2021-05-25 at 10 25 33 AM" src="https://user-images.githubusercontent.com/64283689/119524814-95494e80-bd43-11eb-93fa-e0089027a355.png">

Is there a way to alter the upper and lower limits of the legend for the P2G links when running plotBrowserTrack()?

Thank You!

**Comment by rcorces** [comment]
unfortunately this is hardcoded:
https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/ArchRBrowser.R#L1534

Though the hardcoded default seems logical to me and I dont think most users would need anything different. As you can imagine, with so many moving parts in the plotBrowserTrack function, it is hard to provide so much flexibility.

**Reply by rcorces** [reply]
I guess you could change the loop set that you are passing which would also change the scale. The scale is based off the min and max values of the loopset.

**Comment by sylestiel** [comment]
Awesome. Thanks!
